.github/workflows/pytest_codecov.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.7]
+        python-version: [3.8]
```

Comment:
Why is the python version for tests being changed here? Is this necessary for `gtdb_to_taxdump` installation?

Reply:
It was giving an error with `3.7`, [link](https://github.com/KwanLab/Autometa/runs/7440617864?check_suite_focus=true#step:9:28). `from typing import Union, List, Literal` is only supported in versions >=3.8